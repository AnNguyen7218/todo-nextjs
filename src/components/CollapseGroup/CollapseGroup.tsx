import React from 'react';
import CollapseItem from 'components/CollapseItem/CollapseItem';

type Props = {
  collapseItems: FaqCollapseItemType[];
};

const CollapseGroup: React.FC<Props> = ({ collapseItems }) => {
  return (
    <>
      {collapseItems.map((item, index) => {
        return <CollapseItem item={item} key={index} />;
      })}
    </>
  );
};

export default CollapseGroup;
