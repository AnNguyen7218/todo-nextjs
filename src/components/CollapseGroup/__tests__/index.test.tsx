import React from 'react';
import { render, cleanup } from '@testing-library/react';
import CollapseGroup from '../CollapseGroup';

afterEach(cleanup);

const items = [
  {
    title: 'title',
    content: 'content',
  },
  {
    title: 'title2',
    content: 'content2',
  },
];

describe('CollapseGroup Component', () => {
  test('CollapseGroup should match snapshot', () => {
    const { container } = render(<CollapseGroup collapseItems={items} />);
    expect(container).toMatchSnapshot();
  });
});
