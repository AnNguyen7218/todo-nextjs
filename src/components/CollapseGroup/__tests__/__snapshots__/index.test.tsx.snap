// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollapseGroup Component CollapseGroup should match snapshot 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="header-container"
    >
      <h3
        class="title"
      >
        title
      </h3>
      <ic-add.svg
        classname="icon"
      />
    </div>
    <div
      class="content"
      style="height: 0px; margin-top: 0px;"
    >
      <p>
        content
      </p>
    </div>
  </div>
  <div
    class="container"
  >
    <div
      class="header-container"
    >
      <h3
        class="title"
      >
        title2
      </h3>
      <ic-add.svg
        classname="icon"
      />
    </div>
    <div
      class="content"
      style="height: 0px; margin-top: 0px;"
    >
      <p>
        content2
      </p>
    </div>
  </div>
</div>
`;
